### PR TITLE
Release/beta2 2nd try

### PR DESCRIPTION
**Pull request by oleurud**

### CODE REVIEW

**Comment by andy-esch** [comment]
The binder requirements.txt line for cartoframes should be updated to the version that we're releasing:
https://github.com/CartoDB/cartoframes/blob/develop/binder/requirements.txt#L1

In this case, changing that first line to `cartoframes==1.0b2` is all that's needed. This ensures that the correct version is used when we link to binder with that tag

File: NEWS.rst [hunk 1]
```diff
@@ -8,13 +8,42 @@ Release 2019-xx-xx
...
-- Deprecate cartoframes.analysis.Table in favor of Dataset (#587)
-- Strategy pattern, factory pattern and dynamic sources in Dataset (#834)
+- New Data Observatory API (#806)
+- Widgets
+  - Define Widget API (#809)
+  - Add Histogram Widget (#810)
+  - Add Category Widget (#811)
+  - Add Animation Control Widget (#812)
+  - Add Time Series Widget (#813)
+  - heavy_plus_sign Review Widget AP (#827)
```
Comment (andy-esch) [inline NEWS.rst]: These `heavy_plus_sign`s are remnants from the GH prioritization issues I think and don't render here or seem relevant

Comment (oleurud) [reply NEWS.rst]: :scream:

Comment (andy-esch) [reply NEWS.rst]: This line should be `Review Widget API` (instead of AP)

**Pull request merged**